prow/tide/tide.go

```diff
@@ -35,13 +37,20 @@ import (
 	"k8s.io/test-infra/prow/pjutil"
 )
 
+const (
+	statusContext   string = "tide"
+	statusInPool           = "Will consider for merge once all tests are passing."
```

Comment:
This makes it sound like Tide is waiting for the tests to pass even if all the tests are already green.
I don't know if it is actually necessary to mention that the tests need to be passing here. It seems pretty clear to me that tests have to be passing on a PR in order for it to be automatically merged and we will reiterate this at the top of the Tide dashboard once #6017 merges.

Reply:
OK I'll just remove that bit.